src/components/interaction-designer/block-editors/index.ts

```diff
@@ -20,6 +20,7 @@ export * from './MaximumNumericEditor.vue'
 export * from './MinimumChoicesEditor.vue'
 export * from './MinimumNumericEditor.vue'
 export * from './NameEditor.vue'
+export * from './SimplifiedNameEditor.vue'
```

Comment:
minor: let's put it after 'SemanticLabelEditor' to keep alphabetical order :)

Reply:
thnx, addressed in this PR: https://github.com/FLOIP/flow-builder/pull/350/files#diff-57955344a9992dbe73e7ffbeb0d008abf5ad25d5049f7cac11daa226beb252a9R26